scripts/mineBlocks.ts

```diff
@@ -0,0 +1,19 @@
+import hre from "hardhat";
+import { JsonRpcProvider } from "@ethersproject/providers";
+
+let devchainProvider: JsonRpcProvider;
+
+function useLocalhostProvider() {
+  console.log("switching provider");
+  devchainProvider = new hre.ethers.providers.JsonRpcProvider("http://localhost:7545");
+}
+
+async function mineNBlocks(n: number) {
```

Comment:
Note there is already a `mineBlocks` function in `test-ts/utils.ts`. Could it be reused?